src/content/platform-integration/android/call-jetpack-apis.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Calling JetPack APIs"
+description: "Utilize the latest Android APIs from your Dart code"
```

Comment:
```suggestion
description: "Use the latest Android APIs from your Dart code"
```

I really like this description of the difference between "use" and "utilize". Utilize is often misused.  https://eliteediting.com/resources/grammar/use-vs-utilize/#:~:text=The%20definition%20of%20use%20is,an%20assumed%20strategy%20of%20employment.

Reply:
TIL